docs/sp-add-ins/CSOM methods for applying retention labels

```diff
@@ -0,0 +1,83 @@
+---
+author: kyracatwork
+ms.author: kyrachurney
+ms.date: 4/18/2023
+title: CSOM methods for applying retention labels (setting ComplianceTags)
+description: CSOM methods are available to apply (set) a retention label (ComplianceTag) on one or many items (ListItems) in SharePoint. 
+---
+
+# CSOM methods for applying retention labels (setting ComplianceTags)
+
+Retention labels let you apply retention settings for governance control at the item level, and are part of the Microsoft Purview compliance solutions. [Learn more about retention labels.](/microsoft-365/compliance/retention?view=o365-worldwide#retention-labels)
+
+Retention labels may classify contents as records, which place restrictions on what actions are allowed or blocked. [Learn more about declaring records by using retention labels](/microsoft-365/compliance/declare-records)
+
+CSOM methods are available to apply (set) a retention label (ComplianceTag) on one or many items (ListItems) in SharePoint. 
+
+> [!IMPORTANT]
+> Some information relates to prerelease product that may be substantially modified before it’s released. Microsoft makes no warranties, express or implied, with respect to the information provided here.
+
+## SetComplianceTagOnBulkItems
+This method can be used to set a ComplianceTag on one or many ListItems. It is strongly recommended to use this method for this purpose.
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+```c#
+public List<int> SetComplianceTagOnBulkItems( 
+             List<int> itemIds, 
+             string listUrl, 
+             string complianceTagValue)
+```
+
+### Parameters
+'List' [Int](/en-us/dotnet/api/system.int32)
```

Comment:
If this is a list, please convert to bullet list.

Reply:
Actually I just realized there was another interpretation. I'll reach out directly.